core/crypto/hasher/hasher_impl.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "crypto/hasher/hasher_impl.hpp"
+#include "crypto/blake2/blake2b.h"
+#include "crypto/sha/sha256.hpp"
+#include "crypto/twox/twox.hpp"
+#include "gsl/span"
```

Comment:
gsl/span is 3rd party dependency. sort deps to be 
```
#incude <std libs>

#include <3rd party libs>

#include "local project headers"

Reply:
Ok. Moved. However, you forgot exact header of cpp file, it must be placed at first place.